### DISCUSSION TITLE
Huge fps drop during play loved beatmap

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21484] [Q&A] [answered]
This slider in that diff https://osu.ppy.sh/beatmapsets/1746664#osu/3598398 causes huge fps drops while playing and crushes the game in editor when trying to select.
![image](https://user-images.githubusercontent.com/86934170/205098887-50a4cbfe-7004-438b-bd8c-63560f56c3cc.png)

https://user-images.githubusercontent.com/86934170/205100095-c82a7861-5d54-421a-9776-75cef83b43c5.mp4

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21495
